docs/csharp/language-reference/operators/lambda-expressions.md

```diff
@@ -50,7 +50,7 @@ A lambda expression with an expression on the right side of the `=>` operator is
 (input-parameters) => expression
 ```
 
-The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET common language runtime, such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET common language runtime.
+The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET Common Language Runtime(CLR), such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET Common Language Runtime(CLR).
```

Comment:
White spaces are missing before opening parentheses

Reply:
Here's the fix:

```suggestion
The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET Common Language Runtime (CLR), such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET Common Language Runtime (CLR).
```